edubots_surveys.Rnw

```diff
@@ -295,6 +295,7 @@ freq.Discipline %>% mutate(Discipline = ifelse(as.character(Discipline) == "No",
 \section{Introduction}
 
 % TODO: introduction
+The introduction of new technologies in the classroom involves a combination of teacher training, student enthusiasm and possibly acquiring new skills by both. Some technologies are readily adapted, but others require longer time for full adoption. In most cases, the magnitude of the uptake of the new technology impinges on the collaboration of all the parties using it. Hence, examining the users’ attitudes and the opinions is an important first step in boosting this collaboration and reaping the benefits of the technology.  Recently, a surge in the use of synchronous instant messaging tools in higher education such as WhatsApp, Telegram and Facebook Messenger took place. Some messaging applications have built in chatbots that are able to support a synchronous conversation between the different parties \cite{differ.chat}, while other chatbots are developed as standalone systems and can possibly be attached to a messaging application using an API (Application Program Interface). 
```

Comment:
Noorhan, could you please delete whatever this substituted? I think it's still there, in the next paragraph. That way it's easier to compare what's being changed.

Reply:
I have changed deleted the changed paragraph, please check that it's OK.